.codegen/api.go.tmpl

```diff
@@ -159,6 +159,79 @@ func (a *{{.Service.Name}}API) {{.PascalName}}AndWait(ctx context.Context{{if .R
 	return wait.Get()
 }
 {{end}}{{if .Pagination}}
+{{.Comment "// " 80}}
+func (a *{{.Service.Name}}API) {{.PascalName}}Iter(ctx context.Context{{if .Request}}, request {{.Request.PascalName}}{{end}}) func () (*{{.Pagination.Entity.PascalName}}, error) {
+	var err error
+	finished := false
```

Comment:
Mix of `:=` and `var foo`. Use of the same style is better, IMO.

Looking at the generated code, I was confused what `finished` meant because it doesn't break out. A more apt name would be along the lines of `isLastPage`, because `results` can still be non-empty.

Reply:
`err` is uninitialized (gets its zero value) but `finished` is "initialized" to `false`. I thought it was more clear to set this explicitly this way rather than write `var finished bool`. Are you saying it's better to write
```go
var err error
var finished bool
```?

Reply:
I was thinking of:
```go
var finished = false
```

(also see other comment on name)

Reply:
Yeah, I was having trouble naming that parameter. `isLastPage` is great, I'll use that.